app/screens/settings/notification_settings_mobile/notification_settings_mobile.android.js

```diff
@@ -21,8 +22,17 @@ import SectionItem from 'app/screens/settings/section_item';
 import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
 
 import NotificationSettingsMobileBase from './notification_settings_mobile_base';
+import {getNotificationProps} from '../../../utils/notify_props';
+import deepEqual from 'deep-equal';
```

Comment:
Like with my other comment about import order, `deep-equal` and `prop-types` should come at the top, and `getNotificationProps` would come just above the imports from `app/utils/theme`

Reply:
Done.